src/Compilers/CSharp/Test/Emit3/Attributes/AttributeTests_Locations.cs

```diff
@@ -632,24 +632,36 @@ int Goo
 }
 ";
             CreateCompilation(source).VerifyDiagnostics(
+                // (25,9): warning CS9265: The 'set' accessor of property 'C.Goo' should use the backing 'field' because the other accessor is using it.
+                //         set { }
```

Comment:
>set { } [](http://example.com/codeflow?start=27&length=7)

Are we generating a backing field for this property?

Reply:
It looks like #75461 will fix this.